src/com/homework5/firstvariant/Calculator.java

```diff
@@ -0,0 +1,84 @@
+package com.homework5.firstvariant;
+
+import java.util.Scanner;
+
+public class Calculator {
```

Comment:
It is not the realization of the Calculator. It just a hardcode.
You should implement one method. Example:
```
double cal(double x, String operation, double y){
....
}
```
cal(1,"+",3) should return 4 etc.